frontend-shared/styles/mixins/focus.scss

```
@@ -1,3 +1,25 @@
+@use '../variables' as var;
+
+/**
+ * Add stylized focus to interactive elements such <input> or <textarea>
+ *
+ * @param {boolean} $inset -
+ *   Does the outline render inset or not. The focus style is created with a box-shadow and this
+ *   parameter sets the box-shadow to be inset on the element. Set this to true  when the focusable
+ *   element may be adjacent to another and prevent part of the outline from being covered.
```

```suggestion
 *   The focus style is implemented with a box-shadow and this parameter determines whether
 *   the box-shadow should be inset on the element. Set this to true when the element
 *   may be adjacent to another in order to prevent part of the outline from being obscured.
```

Did I understand that correctly ^^ ?

---

Thats more clear, thank you!